Core/src/Utilities/Logger.cpp

```diff
@@ -70,7 +70,8 @@ void setFailureThreshold(Level level) {
 
 void setFailureThreshold(Level) {
   throw std::logic_error{
-      "Compile-time log failure threshold defined, unable to override"};
+      "Compile-time log failure threshold defined (ACTS_LOG_FAILURE_THRESHOLD "
+      "or ACTS_ENABLE_LOG_FAILURE_THRESHOLD are set), unable to override"};
```

Comment:
Well, `ACTS_ENABLE_LOG_FAILURE_THRESHOLD` could be set to "ON".

How about "`ACTS_LOG_FAILURE_THRESHOLD` is set or `ACTS_ENABLE_LOG_FAILURE_THRESHOLD` is OFF"?

Reply:
Ok.